cmake/Modules/FindAWSSDK_EP.cmake

```diff
@@ -69,6 +69,15 @@ if (NOT AWSSDK_FOUND)
       list(APPEND DEPENDS ep_zlib)
     endif()
 
+    # Set AWS cmake build to use specified build type except for gcc
+    # For aws sdk and gcc we must always build in release mode
+    # See https://github.com/TileDB-Inc/TileDB/issues/1351 and
+    # https://github.com/awslabs/aws-checksums/issues/8
+    set(AWS_CMAKE_BUILD_TYPE ${CMAKE_BUILD_TYPE})
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+        set(AWS_CMAKE_BUILD_TYPE "RELEASE")
```

Comment:
Does this work? Maybe CMake does some internal case normalization, but I've only ever seen `Release`, `Debug`, etc. https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html

Reply:
I don't believe case matters but I will change it to `Release` for consistency.